src/lib/mod.rs

```diff
@@ -27,7 +27,8 @@ impl GameOfLife {
         GameOfLife {
             board,
             width: width as usize,
-            height: height as usize
+            height: height as usize,
+            generation: 0u64,
```

Comment:
**If** we're keeping generation, this literal should just be `0` instead of `0u64`.

Reply:
Can't argue with that.